test/index.js

```diff
@@ -75,7 +75,7 @@ describe('RLP encoding (integer):', function () {
   })
 
   it('it should handle zero', function () {
-    assert.equal(RLP.encode(0).toString('hex'), '80')
+    assert.equal(RLP.encode(0).toString('hex'), '00')
```

Comment:
@hamdiallam Can you please add a small comment, maybe mentioning the link to this PR (or #28) and revert the change in `index.js` so we can merge this instead of closing it?

Of course if you're still in doubt about whether `0x80` is the correct encoding we can keep the discussion open.

Reply:
I'm confused as to why the encoding of 0 should be '80', given the input is a single byte.

Inconsistent with this statement in the encoding specification:
`For a single byte whose value is in the [0x00, 0x7f] range, that byte is its own RLP encoding.`

Reply:
I see the comment made being that `0` is an empty byte array. Is this expected behavior? I'd assume `''` or `[]` is an empty byte array but not `0`